neo4j-ruby-driver.gemspec

```diff
@@ -33,32 +33,34 @@ Gem::Specification.new do |spec|
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
 
-  ffi = ENV['SEABOLT_LIB']&.length&.positive?
-  pdir = ffi ? 'ffi' : 'jruby'
+  mri = !RUBY_PLATFORM.match?(/java/)
+  pdir = mri ? 'ffi' : ENV.has_key?('SEABOLT_LIB') ? 'ffi' : 'jruby'
 
   spec.files << Dir['lib/**/*.rb']
   spec.files << Dir["#{pdir}/**/*.rb"]
-  spec.files << Dir["#{pdir}/**/*.jar"]
+
+  unless mri
+    spec.files << Dir["#{pdir}/**/*.jar"]
+    spec.platform = 'java'
+  end
 
   spec.bindir = 'exe'
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib', pdir]
 
-  spec.platform = 'java' if RUBY_PLATFORM.match?(/java/)
 
-  if ffi
-    spec.add_runtime_dependency 'ffi'
-  else
+  if !mri
     spec.add_runtime_dependency 'jar-dependencies'
     spec.requirements << 'jar org.neo4j.driver, neo4j-java-driver, 1.7.5'
     # avoids to install it on the fly when jar-dependencies needs it
     spec.add_development_dependency 'ruby-maven'
+  else
+    spec.add_runtime_dependency 'ffi'
   end
 
   spec.add_runtime_dependency 'activesupport'
-  spec.add_runtime_dependency 'zeitwerk'
+  spec.add_runtime_dependency 'zeitwerk', '= 2.1.6'
```

Comment:
use the ignore line in loader instead of resorting to older zeitwerk version, see result_stream branch